test/setup/variables.tf

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+variable "org_id" {
+  description = "The numeric organization id"
+}
+
+variable "folder_id" {
+  description = "The folder to deploy in"
+}
+
+variable "billing_account" {
+  description = "The billing account id associated with the project, e.g. XXXXXX-YYYYYY-ZZZZZZ"
+}
+
+variable "region" {
+  description = "Compute region to create test resources"
+  default     = "us-east4"
+}
```

Comment:
Need newline at end of file.

Reply:
> Since testing TFV doesn't actually require creating any resources, I don't see a reason to create an ephemeral test project + service account.

Reply:
> Since testing TFV doesn't actually require creating any resources, I don't see a reason to create an ephemeral test project + service account.

Reply:
@kopachevsky As discussed, just use Cloud Build credentials.

Reply:
@morgante added PR for CFT  org-test https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/pull/376